mkdocs.yml

```diff
@@ -0,0 +1,15 @@
+site_name: ddev Documentation
+edit_uri: edit/master/docs/
+copyright: DRUD Technology, LLC
+theme: readthedocs
```

Comment:
I don't think we need our own theme personally. The readthedocs theme is a popular choice of many other projects, and I rather enjoy the presentation it provides.